lib/puppet_x/puppetlabs/scheduled_task/taskscheduler2_task.rb

```diff
@@ -0,0 +1,260 @@
+# This class is used to manage V1 compatible tasks using the Task Scheduler V2 API
```

Comment:
This blurb is not right upon the 2nd commit and should get a minor update to reflect that.